blog/2022-06-02-two-way.mdx

```diff
@@ -0,0 +1,21 @@
+---
+slug: two-way
+title: IaSQL's 2-way binding between a cloud account and a PostgreSQL database
+authors: [depombo, dfellis, aguillenv]
+---
+
+Infrastructure management tools, specifically Infrastructure as Code (IaC) ones, have trained us to think we can only manipulate a cloud account by applying a static infrastructure declaration, or blueprint, which makes it challenging to manage existing cloud infrastructure and reconcile future changes done outside of the IaC tool of choice. [IaSQL](https://iasql.com), like IaC tools, allows creating, updating or deleting infrastructure in the cloud. However, it also makes it possible to import all existing cloud resources. IaSQL also allows importing recent changes done outside IaSQL with the `iasql_sync()` function after the initial 2-way binding is setup. IaC provides a blueprint of your infrastructure, while IaSQL provides the actual representation of your infrastructure. 
+
+### Understand how each infrastructure component plays into the big picture
+
+The infrastructure of a production system can get complex quickly. Infrastructure components can be used by different resources. A change to a cloud resource can have undesired side effects, or outages, in the application layer when the relationships across infrastructure components is not clear enough. Infrastructure changes are less prone to errors when the complete representation of all the relevant resources is in a relational database. Furthermore, debugging for outliers or misconfiguration can be easily done with SQL. Sometimes the issue under the surface is not misconfiguration, but just configuration drift which can happen when a cloud resource has not been updated in a long time.
+
+### Freedom to use any cloud tool
```

Comment:
I can only think in AnyCloud 
![Alt text](https://raw.githubusercontent.com/alantech/anycloud/main/assets/anycloud.png)